docs/source/analyst/example_usage.ipynb

```diff
@@ -21,87 +21,15 @@
    "metadata": {},
    "source": [
     "## Create an access token\n",
-    "In actual usage, this would be done using [FlowAuth](../../flowauth)."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 2,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "import os\n",
-    "import sys\n",
-    "from dotenv import find_dotenv"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 3,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "path_to_utils_module = os.path.join(\n",
-    "    os.path.dirname(find_dotenv()), \"..\", \"integration_tests\", \"tests\"\n",
-    ")\n",
-    "sys.path.insert(0, path_to_utils_module)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 4,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "from datetime import timedelta\n",
-    "from utils import make_token"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 5,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "claims = {\n",
-    "    \"daily_location\": {\n",
-    "        \"permissions\": {\"run\": True, \"poll\": True, \"get_result\": True},\n",
-    "        \"spatial_aggregation\": [\"admin3\", \"admin2\"],\n",
-    "    },\n",
-    "    \"modal_location\": {\n",
-    "        \"permissions\": {\"run\": True, \"poll\": True, \"get_result\": True},\n",
-    "        \"spatial_aggregation\": [\"admin3\", \"admin2\"],\n",
-    "    },\n",
-    "    \"flows\": {\n",
-    "        \"permissions\": {\"run\": True, \"poll\": True, \"get_result\": True},\n",
-    "        \"spatial_aggregation\": [\"admin3\", \"admin2\", \"admin1\"],\n",
-    "    },\n",
-    "    \"location_event_counts\": {\n",
-    "        \"permissions\": {\"run\": True, \"poll\": True, \"get_result\": True},\n",
-    "        \"spatial_aggregation\": [\"admin3\", \"admin2\", \"admin1\"],\n",
-    "    },\n",
-    "}"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 6,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "TOKEN = make_token(\n",
-    "    username=\"testuser\",\n",
-    "    secret_key=\"secret\",\n",
-    "    lifetime=timedelta(days=1),\n",
-    "    claims=claims,\n",
-    ")"
+    "Generate an API access token using [FlowAuth](../../flowauth)."
```

Comment:
This is likely correct, but I had issues with relative links before where sometimes one more level was needed than I thought. Have you been able to check if it works locally? If not we should double-check once the docs are built via CI.

Reply:
That's a good point. It looks like the file `docs/source/flowauth.md` is still present but not actually used in the deployed docs, and at a first glance the info is indeed all in the "Installing FlowAuth" section so this file should probably be removed. I assume eventually we will want separate docs pages for each individual component again (beyond just installation) but for now probably best to link to the page you mentioned.

Reply:
Looks like we have a similar situation for the README in each component directory - they all link to pages which are still present in the docs, but unreachable from the main docs pages.